### ISSUE TITLE
Should close the current tab when no URL specified

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#157]
**Expected behavior**
Should close the current tab when no URL specified

**Actual behavior**
Getting error - `No target with given URL/Title found.`

**Steps to replicate**
* Run the script
```
const { openBrowser, goto, inputField, into, write, toRightOf, comboBox, click,
    openTab,closeTab } = require('taiko');

(async () => {
    try {
        await openBrowser({headless:false});
        await goto("google.com")
        await openTab("https://gauge.org/index.html")
        await closeTab() // closes current tab
    } catch (e) {
        console.error(e);
    } finally {
        closeBrowser();
    }
})();
```
**Version**
```
commit - b15afd9db54135e2c0a78485df0869913cd9a9a4
```
Blocks #108

**Comment by Apoorva-GA** [comment]
Fix should be available in 06b80e6

**Comment by Apoorva-GA** [comment]
Fix should be available in commit 3ff83f3.

**Comment by Apoorva-GA** [comment]
Not replicable.
